ExpressJS/utils/db.js

```diff
@@ -0,0 +1,93 @@
+const fs = require('fs');
+const { Pool } = require('pg');
+
+// Import the sequlize functionality
+const { Users, Messages, Followers } = require('./sequilize');
+const { Sequelize } = require('sequelize');
+
+
+// PostgreSQL
+let pool = new Pool();
+const SCHEMA_FILE_PATH = 'src/utils/schema_postgres.sql';
+const dbConnectionType = process.env.DB_CONNECTION_TYPE;
+if (dbConnectionType === 'dev_db') {
+  pool = new Pool({
+    user: process.env.POSTGRES_USER,
```

Comment:
Is it possible to just pass the URL?? Then we don't need to have this amount of dotenv variable 🦘

Reply:
Not sure. Let's see. Maybe that's what causing the issue at your new branch